### PR TITLE
add some desc for virtual machine istio install

### PR DESCRIPTION
**Pull request by MXuDong**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by MXuDong** [comment]
@kebe7jun What else do I need to do？

**Comment by craigbox** [comment]
owner review. 
/cc @nmittler

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Follow this guide to deploy Istio and connect a virtual machine to it.
 
 1. Create a virtual machine
 1. Set the environment variables `VM_APP`, `WORK_DIR` , `VM_NAMESPACE`,
-and `SERVICE_ACCOUNT`
+and `SERVICE_ACCOUNT` on your cluster
```
Comment (smawson) [inline content/en/docs/setup/install/virtual-machine/index.md:28]: These aren't on your cluster, these are on your machine that you're using to setup the cluster.

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 2]
```diff
@@ -61,15 +61,17 @@ and `SERVICE_ACCOUNT`
...
+1. Create the working directory on your cluster:
 
     {{< text syntax=bash snip_id=setup_wd >}}
     $ mkdir -p "${WORK_DIR}"
     {{< /text >}}
 
 ## Install the Istio control plane
 
-Install Istio and expose the control plane so that your virtual machine can access it.
+If your cluster already installed the Istio control plane, you can skip this step.
```
Comment (smawson) [inline content/en/docs/setup/install/virtual-machine/index.md:72]: Still need to expose the control plane if you already have it installed.

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 3]
```diff
@@ -61,15 +61,17 @@ and `SERVICE_ACCOUNT`
 
     {{< /tabset >}}
 
-1. Create the working directory:
+1. Create the working directory on your cluster:
```
Comment (howardjohn) [inline content/en/docs/setup/install/virtual-machine/index.md:64]: same comment as above, this is local

File: content/en/docs/setup/install/virtual-machine/index.md (hunk 1 above)
Comment (MXuDong) [reply content/en/docs/setup/install/virtual-machine/index.md:28]: Ok :)

File: content/en/docs/setup/install/virtual-machine/index.md (hunk 2 above)
Comment (MXuDong) [reply content/en/docs/setup/install/virtual-machine/index.md:72]: Ok :)

File: content/en/docs/setup/install/virtual-machine/index.md (hunk 3 above)
Comment (MXuDong) [reply content/en/docs/setup/install/virtual-machine/index.md:64]: Thanks

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 4]
```diff
@@ -61,15 +61,17 @@ and `SERVICE_ACCOUNT`
...
+1. Create the working directory on your machine that you're using to setup the cluster:
 
     {{< text syntax=bash snip_id=setup_wd >}}
     $ mkdir -p "${WORK_DIR}"
     {{< /text >}}
 
 ## Install the Istio control plane
 
-Install Istio and expose the control plane so that your virtual machine can access it.
+If your cluster already installed the Istio control plane, you can skip installation steps, but still need to expose the control plane.
```
Comment (smawson) [inline content/en/docs/setup/install/virtual-machine/index.md:72]:
```suggestion
If your cluster already has an Istio control plane, you can skip the installation steps, but will still need to expose the control plane for virtual machine access.
```

**Pull request merged**